src/main/java/hu/zmesko/Appointment/controller/UserController.java

```diff
@@ -0,0 +1,38 @@
+package hu.zmesko.Appointment.controller;
+
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import hu.zmesko.Appointment.model.AuthRequest;
+import hu.zmesko.Appointment.security.service.JwtService;
+import lombok.RequiredArgsConstructor;
+
+
+@RestController
+@RequestMapping("/auth")
+@RequiredArgsConstructor
+public class UserController {
+
+    private final JwtService jwtService;
+
+    private final AuthenticationManager authenticationManager;
+
+    @PostMapping("/login")
+    public String authenticateAndGetToken(@RequestBody AuthRequest authRequest) {
+        Authentication authentication = authenticationManager.authenticate(
+            new UsernamePasswordAuthenticationToken(authRequest.getUsername(), authRequest.getPassword())
+        );
+        if (authentication.isAuthenticated()) {
```

Comment:
You won't need this check. If you check the [official documentation](https://docs.spring.io/spring-security/reference/api/java/org/springframework/security/authentication/AuthenticationManager.html#authenticate(org.springframework.security.core.Authentication)), it says that if there is some issue with auth, `AuthenticationException` is thrown